test/utils/helpers/fetch.helper.mjs

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable quotes */
+import { API_URL } from '../../../src/data/url.mjs';
+import fetch from 'node-fetch';
+// eslint-disable-next-line camelcase
+import { API_Token } from '../token.mjs';
+
+export const sendRequest = async(url, data = null, method = "get") => {
+    try {
+        const response = await fetch(`${API_URL}${url}`, {
+                  method: method.toUpperCase(),
+                  headers: {
+                      'Content-Type': 'application/json',
+                      // eslint-disable-next-line camelcase
+                      'Authorization': `Bearer ${API_Token}`,
+      
+                  },
+                  body: data ? JSON.stringify(data) : null
+      });
+      const jsonData = await response.json();
+      return {
```

Comment:
I'm slightly confused by the current implementation. The 'sendRequest' function returns an object with 'status' and 'data', yet the test checks response.status. Shouldn't we directly destructure and use 'status'? For example:
`const { status } = await sendRequest('/v1/superadmin_personal/dashboard', postData, 'post');
expect(status).equals(409);`
Could you elaborate on this approach?